### PR TITLE
Support FUSE on Worker in docker file

### PR DESCRIPTION
**Pull request by apc999**
Add libfuse to Alluxio server dockerfile

### CODE REVIEW

File: integration/docker/Dockerfile [hunk 1]
```diff
@@ -32,7 +32,10 @@ RUN if [ ${ENABLE_DYNAMIC_USER} = "true" ] ; then \
        chmod -R 777 /opt/* ; \
     fi
 
-FROM openjdk:8-jdk-alpine
+# instead of ubuntu:16.04, use alluxio/alluxio-ubuntu:ubuntu1604-customize
+# applies optimizations in libfuse on top of ubuntu:16.04
+# it is based off cheyang/fuse2:ubuntu1604-customize
+FROM alluxio/alluxio-ubuntu:1604-customize
```
Comment (cheyang) [inline integration/docker/Dockerfile:38]: I think there are some differences between debian and alpine. For example, it uses 'apt' in debian while 'apk' in alpine

**Review by cheyang — CHANGES_REQUESTED** [review]

**Comment by cheyang** [comment]
And you also need to install fuse lib in worker dockerfile

File: integration/docker/Dockerfile (hunk 1 above)
Comment (apc999) [reply integration/docker/Dockerfile:38]: commented out

**Comment by apc999** [comment]
> And you also need to install fuse lib in worker dockerfile

In `/libfuse`, as part of image
`alluxio/alluxio-ubuntu:1604-customize` which is based off `cheyang/fuse2:ubuntu1604-customize`

File: integration/docker/Dockerfile [hunk 2]
```diff
@@ -73,6 +85,8 @@ RUN if [ ${ALLUXIO_USERNAME} != "root" ] \
       mkdir -p /journal && \
```
Comment (cheyang) [inline integration/docker/Dockerfile:85]: The command to adduser is also different between debian and alpine.

Comment (ZhuTopher) [reply integration/docker/Dockerfile:85]: @apc999 Maybe this is the problem you're running into?

**Review by ZhuTopher — COMMENTED** [review]
The changes LGTM from what I understand about this image

**Comment by cheyang** [comment]
May I ask how to set mount opts in alluxio worker?

**Review by ZhuTopher — APPROVED** [review]
LGTM from what I can tell

**Comment by apc999** [comment]
> May I ask how to set mount opts in alluxio worker?

by `alluxio.worker.fuse.mount.options`. I will create a docfix PR to tell how to set these in docker

**Comment by apc999** [comment]
Tested this PR in docker and macos . it works now. thanks for your help and reivew @cheyang @ZhuTopher

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**